tests/test_relative_free_energy.py

```diff
@@ -162,6 +163,20 @@ def test_run_hif2a_test_system():
     run_bitwise_reproducibility(mol_a, mol_b, core, forcefield, n_frames=100)
 
 
+def test_steps_per_frames():
+    """Verifies that modifying steps per frames doesn't change result trajectory"""
+    mol_a, mol_b, core = get_hif2a_ligand_pair_single_topology()
+    forcefield = Forcefield.load_from_file(DEFAULT_FF)
+    seed = 2022
+    res = run_vacuum(mol_a, mol_b, core, forcefield, None, 5, seed, n_eq_steps=10, steps_per_frame=2, n_windows=3)
+
+    test_res = run_vacuum(mol_a, mol_b, core, forcefield, None, 2, seed, n_eq_steps=10, steps_per_frame=5, n_windows=3)
+
+    # The last frame from the trajectories should match as num_frames * steps_per_frame are equal
+    for frame, test_frame in zip(res.frames, test_res.frames):
+        np.testing.assert_array_equal(frame[-1], test_frame[-1])
+
```

Comment:
I understand this is a minor suggestion and this PR is probably high priority, but it might be nice to also check that the number of frames is as expected in each case.